### PR TITLE
Bump agnhost to 2.22

### PR DESCRIPTION
**Pull request by r4f4**
Also bump kitten & nautilus to 1.4

Signed-off-by: Rafael Fonseca <r4f4rfs@gmail.com>

**What type of PR is this?**

/kind feature

**What this PR does / why we need it**: bump version of agnhost

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**: NONE

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

### CODE REVIEW

File: test/utils/image/manifest.go [hunk 1]
```diff
@@ -219,8 +219,8 @@ func initImageConfigs() map[int]Config {
 	configs[InvalidRegistryImage] = Config{invalidRegistry, "alpine", "3.1"}
 	configs[IpcUtils] = Config{e2eRegistry, "ipc-utils", "1.0"}
 	configs[JessieDnsutils] = Config{e2eRegistry, "jessie-dnsutils", "1.0"}
-	configs[Kitten] = Config{e2eRegistry, "kitten", "1.0"}
-	configs[Nautilus] = Config{e2eRegistry, "nautilus", "1.0"}
+	configs[Kitten] = Config{e2eRegistry, "kitten", "1.4"}
+	configs[Nautilus] = Config{e2eRegistry, "nautilus", "1.4"}
```
Comment (yselkowitz) [inline test/utils/image/manifest.go:223]: Unfortunately, kitten and nautilus 1.4 haven't been pushed to either e2eRegistry nor promoterE2eRegistry

**Comment by r4f4** [comment]
Dropped the kitten and nautilus changes.

**Comment by listx** [comment]
Should this be 2.22 instead? Compare: https://github.com/kubernetes/kubernetes/commit/3e2cb2922b9a81d2846f781cceaf5ee44e39b79e

**Comment by yselkowitz** [comment]
2.21 was correct when this was filed, but if 2.22 is with what 1.19 went stable, then yeah this should match.

**Comment by mkumatag** [comment]
> 2.21 was correct when this was filed, but if 2.22 is with what 1.19 went stable, then yeah this should match.

lets update to 2.22 version

**Comment by mkumatag** [comment]
> > 2.21 was correct when this was filed, but if 2.22 is with what 1.19 went stable, then yeah this should match.
> 
> lets update to 2.22 version

@listx seems like agnhost:2.22 isn't promoted from staging to production, can you please gimme the way to find out a image sha so that I can submit a PR in the k8s.io to promote the image to production?

**Pull request closed without merging**